.github/workflows/test_integration_cli.yml

```diff
@@ -123,6 +125,7 @@ jobs:
             "3.8",
             "3.9",
             "3.10",
+            "3.11",
```

Comment:
Remove?

Reply:
It isn't supported yet, but I am leaving open the option to easily test it.